astropy/table/column.py

```diff
@@ -1230,7 +1237,9 @@ def __setitem__(self, index, value):
 
         # update indices
         self.info.adjust_indices(index, value, len(self))
-        ma.MaskedArray.__setitem__(self, index, value)
+        with warnings.catch_warnings():
+            warnings.simplefilter('ignore', MaskedArrayFutureWarning)
```

Comment:
With the alias of `MaskedArrayFutureWarning = FutureWarning` for the case of no `MaskedArrayFutureWarning`, could this code hide a valid `FutureWarning`?  Or is that not possible?